calculate_average_davecom.sh

```diff
@@ -17,4 +17,5 @@
 
 
 JAVA_OPTS=""
+sdk use java 21.0.1-graal 1>&2
```

Comment:
Could you please move this to a separate file _prepare_davecom.sh_ (see baseline pair of scripts for an example)? It's needed due to a slight reorg to the way we run the entries. Thx!

Reply:
Sure, I just did that and this pull-request is now updated. Thank you.